cpp/include/cudf/types.hpp

```diff
@@ -190,6 +193,7 @@ enum type_id {
   TIMESTAMP_NANOSECONDS,   ///< duration of nanoseconds since Unix Epoch in int64
   DICTIONARY32,            ///< Dictionary type using int32 indices
   STRING,                  ///< String elements
+  LIST,                    ///< List elements
```

Comment:
Do we plan to add Fixed Width lists? If so, should this be named differently?

Reply:
Can you clarify: Is a fixed-width list a list of fixed-width types or a list of fixed length? (If the latter, then I recommend using "fixed-length"). 

What is the purpose?

Reply:
I think he's referring to "Fixed-size" lists, meaning "each row is a list of type X elements of size M".  Eg, each row is 4 integers.

Reply:
So what's the answer to Jake's question?

Reply:
I think the answer is :  probably?  I believe  @kkraus14 thought it would be useful for Dask.